src/components/ui/AudioWaveform.tsx

```
@@ -0,0 +1,358 @@
+        ref={containerRef}
+        onClick={handleContainerClick}
+        onMouseMove={handleMouseMove}
+        onMouseLeave={handleMouseLeave}
+        className="relative w-full h-24 bg-background/40 border border-border/80 rounded-xl cursor-pointer overflow-hidden hover:border-border transition-colors duration-200"
+      >
+        {/* Single Layer: Wavesurfer Dual-Wave Rendering & EDL Overlays via SVG */}
+        <svg
+          className="w-full h-full p-3 overflow-visible"
+          viewBox={`0 0 ${SVG_WIDTH} ${SVG_HEIGHT}`}
+          preserveAspectRatio="none"
+        >
```

Biome (2.4.15)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)